src/app/children/children.service.ts

```diff
@@ -189,6 +190,15 @@ export class ChildrenService {
     );
   }
 
+  getHealthChecksOfChild(childId: string): Observable<HealthCheck[]>{
```

Comment:
this won't scale well with a larger database as it is loading every health record instead of using an index. But as we'll hopefully have a generic solution for this soon, I think this is alright here.